android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt

```diff
@@ -261,11 +264,28 @@ class AddonsManagerAdapter(
         }
 
         holder.allowedInPrivateBrowsingLabel.isVisible = addon.isAllowedInPrivateBrowsing()
-        style?.maybeSetPrivateBrowsingLabelDrawale(holder.allowedInPrivateBrowsingLabel)
+        style?.maybeSetPrivateBrowsingLabelDrawable(holder.allowedInPrivateBrowsingLabel)
 
         fetchIcon(addon, holder.iconView)
         style?.maybeSetAddonNameTextColor(holder.titleView)
         style?.maybeSetAddonSummaryTextColor(holder.summaryView)
+
+        if (addon.isDisabledAsBlocklisted()) {
+            holder.statusBlocklistedView.findViewById<TextView>(R.id.add_on_status_blocklisted_message).text =
+                context.getString(
+                    R.string.mozac_feature_addons_status_blocklisted,
+                    addonName,
+                )
+            holder.statusBlocklistedView.findViewById<TextView>(
+                R.id.add_on_status_blocklisted_learn_more_link,
+            ).setOnClickListener {
+                addonsManagerDelegate.onLearnMoreLinkClicked(
+                    AddonsManagerAdapterDelegate.LearnMoreLinks.BLOCKLISTED_ADDON,
+                    addon,
+                )
+            }
+        }
+        holder.statusBlocklistedView.isVisible = addon.isDisabledAsBlocklisted()
```

Comment:
Maybe we should move this statement to be inside the `if` block. If it's already there sorry 😅